docs/user-manual/en/client-reconnection.md

```diff
@@ -89,9 +89,20 @@ Client reconnection is configured using the following parameters:
   ridiculously large values. By setting this parameter you can set an upper limit
   on that value. The default value is `2000` milliseconds.
 
+- `ha`. This optional parameter determines weather the client will try to
+  reconnect to the backup node when the live node is not reachable.
+  The default value is `false`.
+  For more information on HA, please see [High Availability and Failover](ha.md).
+
 - `reconnectAttempts`. This optional parameter determines the total number of
-  reconnect attempts to make before giving up and shutting down. A value of
-  `-1` signifies an unlimited number of attempts. The default value is `0`.
+  reconnect attempts to make to the current live/backup pair before giving up.
```

Comment:
I am confused :-)

I thought failover was between live and backup

reconnect should first try to failover, then fall back to any connection to eventually reconnect. these seems the opposite.

Reply:
Yeah, could probably do with some docs to clarify what specifically is a 'reconnect attempt' vs a 'failover attempt' and thus help show when you would use one/other/both, for those that dont already know the previous/existing behaviour and thus also wont be able tell later which one of these is the new option/behaviour (/me raises hand).

(E.g, in other clients I know, there is only one notion of 'attempts', where I believe generally they retry the server originally connected to again and then try all the other servers currently known about until succeeding, then repeat [after delay] if still not connected...with that one-shot-at-each-server process collectively being '1 attempt')

Reply:
@gtully @gemmellr thanks for your feedback, I'll review the doc and I'll move the pause after trying all available connectors.

Reply:
done